core/src/main/java/org/testcontainers/containers/Container.java

```diff
@@ -383,6 +408,7 @@ default Integer getFirstMappedPort() {
      */
     SELF withLogConsumer(Consumer<OutputFrame> consumer);
 
+    @Deprecated
```

Comment:
Is this related to the change? Same question for deprecation `getContainerInfo()`.

I don't mind deprecating these, as it makes things cleaner, but would it be possible to have a comment indicating a recommended alternative approach?